lib/backburner.js

```diff
@@ -73,6 +78,59 @@ Backburner.prototype = {
     }
   },
 
+  /**
+   Trigger an event. Supports up to two arguments. Designed around
+   triggering transition events from one run loop instance to the
+   next, which requires an argument for the first instance and then
+   an argument for the next instance.
+
+   @private
+   @method _trigger
+   @param {String} eventName
+   @param {any} arg1
+   @param {any} arg2
+   */
+  _trigger: function(eventName, arg1, arg2) {
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
+      for (var i = 0; i < callbacks.length; i++) {
+        callbacks[i].call(null, arg1, arg2);
+      }
+    }
+  },
+
+  on: function(eventName, callback) {
+    if (typeof callback !== 'function') {
+      throw new TypeError('Callback must be a function');
+    }
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
```

Comment:
if this is false, we should error (someone pushing something onto a non-existent event